tooltester/src/main/java/io/dockstore/tooltester/runWorkflow/WorkflowList.java

```diff
@@ -0,0 +1,50 @@
+/*
+ *    Copyright 2023
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package io.dockstore.tooltester.runWorkflow;
+
+import io.dockstore.openapi.client.api.ExtendedGa4GhApi;
+import io.dockstore.openapi.client.api.Ga4Ghv20Api;
+
+import java.util.List;
+
+/**
+ * Contains a list of workflows that will be tested when "run-workflows" is ran
+ */
+public class WorkflowList {
+
+    private Ga4Ghv20Api ga4Ghv20Api;
+    private ExtendedGa4GhApi extendedGa4GhApi;
+    public WorkflowList(Ga4Ghv20Api ga4Ghv20Api, ExtendedGa4GhApi extendedGa4GhApi) throws InterruptedException {
+        this.ga4Ghv20Api = ga4Ghv20Api;
+        this.extendedGa4GhApi = extendedGa4GhApi;
+        this.workflowsToTest = getDefaultTests();
+    }
+    private final List<WorkflowRunner> workflowsToTest;
+
+    private List<WorkflowRunner> getDefaultTests() throws InterruptedException {
+        return List.of(new WorkflowRunner("github.com/dockstore-testing/wes-testing/agc-fastq-read-counts", "main", "test-parameter-files/agc-fastq-read-counts-test-parameter-file.json", extendedGa4GhApi),
```

Comment:
I'm a little confused how this works, perhaps update the JavaDoc in WorkflowRunner?

* This JSON file that you have locally, elsewhere in the commit
* The next line references a JSON file that looks to be on qa.dockstore.org, which jibes with the JavaDoc for that constructor: `The relative path to the test parameter file on the dockstore site (ex. agc-examples/fastq/input.json)`

I'm probably missing something obvious, but can you please clarify?

Reply:
My bad, realize that I didn't change the javadoc for one of the constructors. It has now been updated.

The idea for the `WorkflowRunner` is to have two different constructors, one that uses the test-parameter file found on dockstore (such is the case [here](https://github.com/dockstore/dockstore-support/blob/114370ceff15c229ce5e15243682908debde00db/tooltester/src/main/java/io/dockstore/tooltester/runWorkflow/WorkflowList.java#L40)), and one that use a local test-parameter file (such is the case [here](https://github.com/dockstore/dockstore-support/blob/114370ceff15c229ce5e15243682908debde00db/tooltester/src/main/java/io/dockstore/tooltester/runWorkflow/WorkflowList.java#L42)).

This is necessary because very very few workflows on dockstore use `S3` file names in their test-parameter file. So at least for testing (although potentially for future use too), I found it necessary to have the ability to use local test parameter files, instead of just those found on dockstore.